From 5496789266406da0b14db0e874fb48027769d37d Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 23 Aug 2017 09:33:15 +0300
Subject: [PATCH 1184/1345] fix: dts: arm64: marvell: Remove Comphy handlers
 from SATA nodes

commit  0c2a2ec855259becae6b2ac66277ca4b74a1854d from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Comphy is disabled in vanilla device trees, comphy entries
should be removed from SATA nodes.
Patch fix broken SATA on vanilla device trees.

Change-Id: Ib913cd364c33fec2ac4125f0f6ea5710d1ce29d9
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43324
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/marvell/armada-7040-db-pp2-vanilla.dts     |    2 --
 .../dts/marvell/armada-8040-db-pp2-vanilla.dts     |    8 --------
 .../dts/marvell/armada-8040-mcbin-pp2-vanilla.dts  |    6 ------
 3 files changed, 16 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
index ac0e02f..10b5ab2 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
@@ -99,8 +99,6 @@
 					status = "disabled";
 				};
 				sata-port@1 {
-					phys = <&comphy0 3 COMPHY_SATA1>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 			};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
index cc9610f..ea8e394 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
@@ -87,13 +87,9 @@
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
-					phys = <&comphy0 1 COMPHY_SATA0>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 				sata-port@1 {
-					phys = <&comphy0 3 COMPHY_SATA1>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 			};
@@ -197,13 +193,9 @@
 			sata@540000 {
 				status = "okay";
 				sata-port@0 {
-					phys = <&comphy1 1 COMPHY_SATA0>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 				sata-port@1 {
-					phys = <&comphy1 3 COMPHY_SATA1>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 			};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
index 9fa94f6..2a478665 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
@@ -125,8 +125,6 @@
 					status = "disabled";
 				};
 				sata-port@1 {
-					phys = <&comphy0 5 COMPHY_SATA1>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 			};
@@ -249,13 +247,9 @@
 				status = "okay";
 
 				sata-port@0 {
-					phys = <&comphy1 1 COMPHY_SATA0>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 				sata-port@1 {
-					phys = <&comphy1 3 COMPHY_SATA1>;
-					phy-names = "comphy";
 					status = "okay";
 				};
 			};
-- 
1.7.9.5

